src/components/NewLocationPage/NotesBlock/NotesBlock.style.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+import { COLOR_MAP } from 'common/colors';
+
+export const SectionContentContainer = styled.div`
+  display: flex;
+  font-size: initial;
+  text-transform: none;
+`;
+
+export const TextContainer = styled.div``;
+
+export const TextComponent = styled.p`
+  color: ${COLOR_MAP.GREY_4};
+  font-weight: initial;
```

Comment:
[same as above](https://github.com/covid-projections/covid-projections/pull/3403/files#r618805534). we can remove `font-weight: initial;`